Added Gavin Reid advice page

gjlreid opened the pull request:
New Advice page added along with link to the list of advice page

twosavoie commented:
Hi @gjlreid,

This is great!

There's one small change to make. In your advice file, there's a link to a **Gavin Reid.css** file. But this file isn't in your pull request. Can you either add the Gavin Reid.css file to the css folder or change the link back to the main.css file. Either will work so it's totally up to you!

If you decide to use your own css file, be sure to add a dash between "Gavin" and "Reid". Computers find spaces in file names confusing. 🙂 

Once you've made that change, please add the change with `git add -A`, commit as per usual and push up your advice branch once more, which will automatically update this pull request.

Then we should be all set to merge in your code. Let us know if you have any questions!

Best,
Lisa

twosavoie commented:
Hi @gjlreid,

Excellent question. You'll want to push the advice branch instead of the main branch. Before you do, check to make sure the changes were made on the advice branch. 

Best,
Lisa

twosavoie commented:
Hi @gjlreid,

Just checking to see if you need any help with this pull request. Let us know if you have any questions!

Best,
Lisa

twosavoie commented on advice/Gavin-Reid-advice.html:
```diff
@@ -0,0 +1,54 @@
+
+    <head>
+        <meta charset="utf-8">
+        <link rel="stylesheet" type="text/css" href="../css/Gavin Reid.css">
```
Hi @gjlreid,

You're super close! You added a link to your own CSS file in the <head> of your advice.html file. But the css file you linked to isn't in the css folder. If you've decided not to add one, you could change it back to the main.css file. Either way would be completely fine!

Best,
Lisa

gjlreid commented:
Hi Lisa,

I'm really confused. I did all that. I added a link to my own CSS file in
html and added the file to the css folder. I then pushed the advice
changes, this was 10 days ago. I now can't see any of my work on GitHub.

Do I need to do it all again? Where is the best place to start?
Thanks,
Gavin

On Wed, 23 Aug 2023 at 23:28, Lisa Savoie ***@***.***> wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In advice/Gavin-Reid-advice.html
> <https://github.com/skillcrush/git-practice/pull/1109#discussion_r1303600308>
> :
>
> > @@ -0,0 +1,54 @@
> +<!DOCTYPE html>
> +<html>
> +
> +    <head>
> +        <meta charset="utf-8">
> +        <link rel="stylesheet" type="text/css" href="../css/Gavin Reid.css">
>
> Hi @gjlreid <https://github.com/gjlreid>,
>
> You're super close! You added a link to your own CSS file in the of your
> advice.html file. But the css file you linked to isn't in the css folder.
> If you've decided not to add one, you could change it back to the main.css
> file. Either way would be completely fine!
>
> Best,
> Lisa
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/skillcrush/git-practice/pull/1109#pullrequestreview-1592497047>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BA7VYDBZ4WB66VGJQUKWBZDXWZ7XPANCNFSM6AAAAAA3REDCMQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

twosavoie commented:
Hey @gjlreid,

We haven't heard from you in a while, so we're going to go ahead and close this pull request. Feel free to submit a new one when you're ready!

Best,
Lisa
